<launch>

  <node pkg="vio_gpu" type="vio_gpu" name="vio" clear_params="true" output="screen">

    <!-- Camera topic to subscribe to -->
    <param name="cam_topic" value="/CoreNode/grey_img" type="str" />
    <param name="imu_topic" value="/SensorNode/imu" type="str" />
    
    <!-- Camera calibration file -->
    <rosparam file="$(find vio_gpu)/param/px30.yaml" />

    <!-- Initial camera orientation, make it point downwards -->
    <param name="init_rx" value="0.0" />
    <param name="init_ry" value="-0.785398" /> <!--  -tetha (-35) -->
    <param name="init_rz" value="3.14" /> <!-- 180  -->

  </node>

  <!-- Default parameter settings: choose between vo_fast and vo_accurate -->
  <rosparam file="$(find vio_gpu)/param/vo_fast.yaml" />

</launch>
